src/styles/overlay.scss

```diff
@@ -0,0 +1,8 @@
+.overlay {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background: rgba(0, 0, 0, 0.8);
+}
```

Comment:
Je me demande si on ne pourrait pas mettre le style .overlay aussi dans commun.scss ? J'ai l'impression que je pourrais m'en servir aussi pour l'affichage du contenu.

Reply:
Hmm 🤔 
J'ai l'impression que tu pourrais t'en servir en important `overlay.scss` 😉

J'ai l'impression que `commun.scss` regroupe les styles pouvant s'appliquer à des éléments divers. En l'état, le style lié à `.overlay` reste spécifique à un overlay – et je ne vois pas la motivation à le mettre dans `commun.scss`.

Reply:
Je voudrai l'utiliser pour la div `contenu` qui se retrouve être un overlay dans mon cas mais pas seulement. En effet, j'ai mis l'étiquette comme une sous div de l'overlay. Ca marche bien et ça m'assure que l'étiquette est bien toujours au dessus de l'overlay.

Reply:
by the way, la définition de contenu c'est : 
```
    position: fixed;
    left: 0;
    top: 0;
    width: 100vw;
    height: 100vh;
```

Je pense que c'est nécessaire pour garantir que ton overlay recouvre bien tout l'écran et pas seulement 100% de son parent… Est-ce que c'est une intention dans ton cas de ne pas recouvrir tout l'écran ?

Reply:
Mon intention est de recouvrir toute la zone de la simulation - sans préjuger si elle prendra tout l'écran.

Comme convenu par oral - on reste comme ça pour l'instant.